lib/modules/prayer_times/screens/prayer_times.dart

```diff
@@ -56,26 +54,21 @@ class _PrayerTimesState extends State<PrayerTimes> {
     ]);
   }
 
-  void _onRefresh(
+  Future<void> _onRefresh(
       UserSettingsState userSettings, PrayerTimesBloc prayerTimesBloc) async {
```

Comment:
I think no need to be async, we are not using await

Reply:
Yes, but that's the type LiquidPullToRefresh onRefresh function expects